src/java/org/apache/cassandra/cql3/statements/schema/TableAttributes.java

```diff
@@ -98,6 +98,9 @@ public static Set<String> allKeywords()
 
     private TableParams build(TableParams.Builder builder)
     {
+        if (hasOption(Option.ALLOW_AUTO_SNAPSHOT))
+            builder.allowAutoSnapshot(getBoolean(Option.ALLOW_AUTO_SNAPSHOT.toString(), false));
```

Comment:
I don't think this default value is correct. But more importantly, we know the option is present, so we shouldn't even be passing the default. Need to add a method that doesn't take a default value. And also do the same for `cdc` while here.